### PR TITLE
Integrate search parameters feature reference into search API reference

### PR DESCRIPTION
**Pull request by guimachiavelli**
Part of our ongoing efforts to remove the "Feature reference" section.

This PR moves the content of the "Search parameter" feature reference inside the API search reference.

### CODE REVIEW

**Review by maryamsulemani97 — DISMISSED** [review]
🍕

**Comment by guimachiavelli** [comment]
Sorry, create this in a bit of a rush and completely missed the heading level issue.

Regarding `sidebarDepth`, I don't think our setup supports more than 1 level, so I'm leaving it out of this PR.

**Review by dichotommy — DISMISSED** [review]
LGTM 👾

File: .vuepress/config.js [hunk 1]
```diff
@@ -275,7 +275,6 @@ module.exports = {
           path: '/reference/features/',
```
Comment (maryamsulemani97) [inline .vuepress/config.js:275]: We don’t need this heading now, right? 🤔

**Review by maryamsulemani97 — APPROVED** [review]
🤖

**Comment by guimachiavelli** [comment]
bors merge

**Pull request merged**